src/guacd-docker/bin/link-freerdp-plugins.sh

```diff
@@ -58,26 +57,25 @@ where_is_freerdp() {
     fi
 
     echo "$PATHS"
-
 }
 
 #
 # Create symbolic links as necessary to include all given plugins within the
 # search path of FreeRDP
 #
 
-while [ -n "$1" ]; do
+GUAC_PLUGIN="$1$2"
+FREERDP_DIR="$(where_is_freerdp "$1")"
+FREERDP_PLUGIN_DIR="${FREERDP_DIR}/freerdp2"
 
-    # Determine correct install location for FreeRDP plugins
-    FREERDP_DIR="$(where_is_freerdp "$1")"
-    FREERDP_PLUGIN_DIR="${FREERDP_DIR}/freerdp2"
+for value in $GUAC_PLUGIN; do
```

Comment:
This seems a bit wonky. With `GUAC_PLUGIN` being assigned `"$1$2"`, won't `for value in $GUAC_PLUGIN` always just loop through a single value except in the case where `GUAC_PLUGIN` happens to contain whitespace, in which case it would (presumably undesirably) split that value up?